### PR TITLE
Optimization fixes (v.1)

### PR DESCRIPTION
**Pull request by coconut3-dev**
- heavy assets optimized
 - font injection
 - added lang attribute

### CODE REVIEW

File: app.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html>
-<html {{ HTML_ATTRS }}>
+<html {{ HTML_ATTRS }} lang="en-US">
 <head {{ HEAD_ATTRS }}>
   {{ HEAD }}
   <meta name="google-site-verification" content="KkxplZrJKbimOGuYdsRlAVVimi6dj_ve7tPbibLr8oc" />
+  <link rel="stylesheet" href="https://use.typekit.net/jus7nrc.css">
```
[the remark this answers is not in the document]
Comment (coconut3-dev) [reply app.html:6]: Got it!

[the remark this answers is not in the document]
Comment (coconut3-dev) [reply app.html]: Removed it

**Comment by coconut3-dev** [comment]
I have removed this line

**Comment by cjdelisle** [comment]
Thanks

**Pull request merged**